modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/table-view/EditTableViewModal.es.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayModal, {useModal} from '@clayui/modal';
+import {MODALS} from 'app-builder-web/js/pages/apps/constants.es';
+import EditTableView from 'app-builder-web/js/pages/table-view/EditTableView.es';
+import React, {useContext} from 'react';
+
+import {ModalContext} from '../ModalContextProvider.es';
+
+const EditTableViewModal = () => {
+	const {setVisibleModal, visibleModal} = useContext(ModalContext);
+	const {observer, onClose} = useModal({
+		onClose: () => {
+			setVisibleModal('');
+		},
+	});
+
+	return (
+		<>
+			{visibleModal === MODALS.EDIT_TABLE_VIEW && (
+				<ClayModal center observer={observer}>
+					<ClayModal.Header className="border-0">
+						{Liferay.Language.get('new-table-view')}
+					</ClayModal.Header>
+					<ClayModal.Body>
+						<EditTableView />
+					</ClayModal.Body>
+					<ClayModal.Footer
+						last={
+							<>
+								<ClayButton
+									className="mr-3"
+									displayType="secondary"
+									onClick={onClose}
+									small
+								>
+									{Liferay.Language.get('cancel')}
+								</ClayButton>
```

Comment:
@gabrielpramos How do you intend to call the save function from the EditTableView page on the modal footer?

Reply:
I'll create a compound component EditTableView.Footer and export it to reutilize on modal's footer with just a few modifications to fit mockup without creating any regression bugs.
I can refine this idea with you.